api/client/service.proto

```diff
@@ -2,8 +2,10 @@ syntax = "proto3";
 
 package nighthawk.client;
 
+import "google/protobuf/wrappers.proto";
```

Comment:
Are these new imports still necessary now that we are adding a new proto file?

(wrappers.proto and base.proto)

Reply:
they can be removed indeed, done in 944259382bb86f75316099c256b964de492b48b5